packages/wallet-sdk/src/sign/SignHandler.ts

```diff
@@ -42,6 +41,7 @@ export class SignHandler {
       url: keysUrl ?? CB_KEYS_URL,
       onConfigUpdateMessage: this.handleConfigUpdateMessage.bind(this),
     });
+    this.signer = this.loadSigner();
```

Comment:
is there any issue with sequence of these lines?

Reply:
yea, loadSigner reads this.metadata, etc, which was undefined